core/packet.h

```diff
@@ -60,6 +60,8 @@ static_assert(SNBUF_SCRATCHPAD_OFF == 320,
 
 namespace bess {
 
+const int minimum_try = 16384; // For packet buffer allocation.
```

Comment:
According to the Google style guide it should be renamed something like `kMinimumTry`. However, I have a different suggestion:
* This `minimum_try` stays in the `init_mempool_socket()`
* `ValidateBuffersPerSocket()` does not check if the value is no less than `minimum_try`.
* `init_mempool_socket()` remains the same. The resulting behavior is that the user specified value will be tried once, but not any further if it is less than `minimum_try`.

Reply:
Will do.